### DISCUSSION TITLE
Eject on native mode on OLED display

### DISCUSSION THREAD

**Discussion by dshadoff** [keirf/flashfloppy#773] [General]
I recently purchased a newer Gotek with the Artery 435 microcontroller and factory OLED and rotary controller (it came with Flash Floppy installed by the seller). As such I am using it in "Native mode".  In this mode, it seems to automatically mount whatever image is currently indexed.

Sometimes this is unproductive - for example, one reboot cycle, you wish to boot from floppy, but the next cycle you don't, so it would be good to unmount the image... that is to say, to have the drive recognized as empty during the boot sequence. But there are warnings against removing the USB stick altogether, so a soft unmount would be the best option.

I understand that the FF.CFG file allows the user to govern mount/unmount behaviour, but going "full manual mount" seems equally tedious, particularly when installing operating systems from multi-floppy sets, with frequent disc changes.

Maybe there is a way to unmount the image when in native mode - and I'm just not aware of it - but if not, it might be nice to have a "no disc" option in the image rotation selection sequence, so that choosing an empty floppy is just as easy as choosing any specific disc image.

**Comment by keirf** [comment]
If you wish to disable autoselect then you can set the two autoselect-* options in FF.CFG to 0. Selecting manually is as simple as pressing the rotary encoder so it is not very onerous and arguably less confusing than auto select.

If you wish to not mount automatically at power on then use the ejected-on-startup option in FF.CFG.

Finally, you can eject mounted image by pressing the encoder at any time.

**Comment by dshadoff** [comment]
> Finally, you can eject mounted image by pressing the encoder at any time.

Thanks, I had previously seen "Eject Menu" appear, but didn't realize that the state was already ejected at that point; I was searching for an option to eject within that menu.
